.github/workflows/local-testnet.yml

```diff
@@ -34,7 +34,6 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
       - name: Install lighthouse
-        if: steps.cache-cargo.outputs.cache-hit != 'true'
```

Comment:
Because of removing this line, [the step for `actions/cache`](https://github.com/sigp/lighthouse/blob/8c7e5ab046b54d085949bd80ee932ec99a86bae4/.github/workflows/local-testnet.yml#L24-L34) can be removed too. 💡

Reply:
I think by removing the `if` and keeping the cache we can rebuild Lighthouse using the cache when we get a cache hit, which should be faster than building from scratch. In this case, we got a cache hit on the `Cargo.lock` but @AgeManning had added a new flag to `lighthouse bn` which didn't get compiled.

Reply:
The flag that was added was actually in this PR: https://github.com/sigp/lighthouse/pull/3023

Reply:
Oh, I understand! Thank you for your explanation.